src/cli.js

```diff
@@ -46,11 +46,18 @@ if (process.env.NO_CLI !== 'true') {
       type: 'string',
       demandOption: false,
     })
+    .options('a', {
+      alias: 'auth',
+      describe: 'Enable authentication and authorization',
+      type: 'boolean',
+      default: false,
+      demandOption: false,
+    })
     .options('S', {
       alias: 'studio',
       describe: 'Start Soul Studio in parallel',
       type: 'boolean',
-      demandOption: false
```

Comment:
@AbegaM - I don't think we require the trailing comma on line #60

Reply:
This issue is also fixed, i have removed all of the trailing commas